packages/yourloops/test/integration/assert/trends.ts

```diff
@@ -81,7 +80,6 @@ export const checkTrendsStatsWidgetsTooltips = async () => {
   await checkStatTooltip(patientStatistics, 'Avg. Daily Time In Range', TIME_IN_RANGE_TOOLTIP)
   await checkStatTooltip(patientStatistics, 'Avg. Glucose (CGM)', AVG_GLUCOSE_TOOLTIP)
   await checkStatTooltip(patientStatistics, 'Sensor Usage', SENSOR_USAGE_TOOLTIP)
-  await checkStatTooltip(patientStatistics, 'GMI (estimated HbA1c)', GMI_TOOLTIP)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏How do you justify removing this test?

Reply:
rtain amount of data. So I'm going to rewrite some tests to be able to correct this

Reply:
If this no longer works, it is a sign of a bug... So are you sure that you don't have a bug?

Reply:
I managed to reintegrate the test and it works

Reply:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏You say that you reintegrated the test but this line is still deleted... :thinking: 
I suppose that you reintegrated it somewhere else? But it should work as before here, right?